.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+version: 2
+updates:
+# Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

```suggestion
      interval: "monthly"
    groups:
      actions:
        patterns:
          - "*"
    labels:
      - "github-actions"
```


Grouping the GitHub actions Dependabot updates is super useful.

While I usually do weekly updates here I might suggest "monthly" just to reduce the maintenance burden on a single person.